docs/simulate/qvm_basic_example.ipynb

```diff
@@ -0,0 +1,432 @@
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "vGUeqO9jn1vc"
+   },
+   "source": [
+    "<table class=\"tfo-notebook-buttons\" align=\"left\">\n",
+    "  <td>\n",
+    "    <a target=\"_blank\" href=\"https://quantumai.google/cirq/simulate/qvm_basic_example\"><img src=\"https://quantumai.google/site-assets/images/buttons/quantumai_logo_1x.png\" />View on QuantumAI</a>\n",
+    "  </td>\n",
+    "  <td>\n",
+    "    <a target=\"_blank\" href=\"https://colab.research.google.com/github/quantumlib/Cirq/blob/master/docs/simulate/qvm_basic_example.ipynb\"><img src=\"https://quantumai.google/site-assets/images/buttons/colab_logo_1x.png\" />Run in Google Colab</a>\n",
+    "  </td>\n",
+    "  <td>\n",
+    "    <a target=\"_blank\" href=\"https://github.com/quantumlib/Cirq/blob/master/docs/simulate/qvm_basic_example.ipynb\"><img src=\"https://quantumai.google/site-assets/images/buttons/github_logo_1x.png\" />View source on GitHub</a>\n",
+    "  </td>\n",
+    "  <td>\n",
+    "    <a href=\"https://storage.googleapis.com/tensorflow_docs/Cirq/docs/simulate/qvm_basic_example.ipynb\"><img src=\"https://quantumai.google/site-assets/images/buttons/download_icon_1x.png\" />Download notebook</a>\n",
+    "  </td>\n",
+    "</table>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "cd6e59ef3edd"
+   },
+   "source": [
+    "This notebook walks through running a simple circuit on the [Quantum Virtual Machine](/cirq/simulate/quantum_virtual_machine), including the necessary constraints on a device-runnable circuit and how to satisfy them. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "Lfira0gPf0Gd"
+   },
+   "source": [
+    "## **Install** Cirq and qsim"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "cellView": "form",
+    "id": "zs5J6wAXqvtW"
+   },
+   "outputs": [],
+   "source": [
+    "# @title Install `cirq_google` and `qsimcirq`\n",
+    "\n",
+    "try:\n",
+    "    import cirq\n",
+    "    import cirq_google\n",
+    "except ImportError:\n",
+    "    print(\"installing cirq...\")\n",
+    "    !pip install --quiet cirq-google\n",
+    "    print(\"installed cirq.\")\n",
+    "    import cirq\n",
+    "    import cirq_google\n",
+    "\n",
+    "try:\n",
+    "    import qsimcirq\n",
+    "except ImportError:\n",
+    "    print(\"installing qsimcirq...\")\n",
+    "    !pip install --quiet qsimcirq\n",
+    "    print(f\"installed qsimcirq.\")\n",
+    "    import qsimcirq\n",
+    "\n",
+    "# Other modules used in this colab\n",
+    "import matplotlib.pyplot as plt\n",
+    "import time"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "p2JAfQa8gVSe"
+   },
+   "source": [
+    "## Create a **Quantum Virtual Machine**\n",
+    "\n",
+    "The following cell builds a Quantum Virtual Machine that mimics a particular Google quantum hardware device (currently Rainbow or Weber), the \"long way\": \n",
```

Comment:
Optional: Prefer replacing the "long way" with a more descriptive phrase, such as "by using individual commands, so that each aspect of the virtual machine can be customized as desired."